Fixing format of the article #909 (#3702)
MicrosoftDocs/windows-itpro-docs: windows/deployment/upgrade/windows-error-reporting.md

ms.topic: article
---

# Windows Error Reporting

**Applies to**
-   Windows 10

>[!NOTE]
> This is a 300 level topic (moderately advanced).  
> See [Resolve Windows 10 upgrade errors](resolve-windows-10-upgrade-errors.md) for a full list of topics in this article.

When Windows Setup fails, the result and extend code are recorded as an informational event in the Application log by Windows Error Reporting as event 1001. The event name is **WinSetupDiag02**.  You can use Event Viewer to review this event, or you can use Windows PowerShell.

To use Windows PowerShell, type the following commands from an elevated Windows PowerShell prompt:

>[!IMPORTANT]

> >}The following source will be available only if you have done an update from a previous version of windows 10 into a new version. If you installed the current version and have not done any update, the source named **WinSetupDiag02** will be unavailable.

Reviewer: ```suggestion
>}The following source will be available only if you have updated from a previous version of windows 10 to a new version. If you installed the current version and have not updated, the source named **WinSetupDiag02** will be unavailable.
```
(This suggestion is mainly based on my daily experience with Windows Update in Windows 10. There could be reasons, unbeknownst to me, to write the sentences different from my suggestion.)
